### DISCUSSION TITLE
Search within a list

### DISCUSSION THREAD

**Discussion by Tobi-De** [community/community#67616] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

As simple as the title suggests, the ability to filter or search the lists (ordering too would be cool). I have some lists that are quite long and it's annoying to have to go through them every time I search for something in a list.

**Comment by shannonmatloob** [comment]
You can order the lists on your profile's [stars tab](https://github.com/Tobi-De?tab=stars), but the sort goes away when you refresh.

**Comment by Tithapour** [comment]
Product feedback body

**Comment by Tithapour** [comment]
- [ ]

**Comment by slmkhanahmed** [comment]
https://addons.mozilla.org/en-US/firefox/addon/git-star/
